titan-compiler/parser.lua

```diff
@@ -135,7 +135,9 @@ function defs.exp_is_call(_, pos, exp)
     end
 end
 
--- Add libraries to defs
+function defs.name2decl(name)
+    return ast.Decl_Decl(name, false)
+end
```

Comment:
If we change the for loop syntax to allow an optional type annotation then we can get rid of this.